.travis.yml

```diff
@@ -3,11 +3,17 @@ dist: trusty
 
 php:
     - '7.4'
+    - nightly
 
 matrix:
     include:
         - php: '7.4'
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+        - php: nightly
+          env: 'COMPOSER_FLAGS="--ignore-platform-reqs"'
```

Comment:
We should avoid this.

Reply:
Hi @GrahamCampbell I agree, this is just while not all deps support PHP8, I'll remove it from the PR at a later point :-)

Reply:
Which deps don't support PHP 8?

Reply:
Just removed as I checked all deps already support PHP 8 :-D